libs/core/algorithms/include/hpx/parallel/algorithms/uninitialized_relocate.hpp

```diff
@@ -0,0 +1,274 @@
+//  Copyright (c) 2014-2023 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+/// \file parallel/algorithms/uninitialized_relocate.hpp
+
```

Comment:
Would you mind adding some docs as well? @dimitraka can help with the integration into the documentation build etc.